src/basic-syntax/static-and-const.md

```diff
@@ -39,9 +39,11 @@ fn main() {
 ```
 
 As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and the variable lives through the entirety of the program execution.
+When a globally-scoped value does not have a reason to need its own address in memory, `const` is generally preferred.
```

Comment:
... and doesn't have interior mutability. E.g. making a static `Mutex<MyType>` into a const wouldn't work.

Reply:
Would saying "object identity" instead of "its own address in memory" capture both of these requirements, and then we could specify "(e.g. requiring its own address in memory or its own dynamic state such as types like `Mutex<T>` which have interior mutability)"? Or is that too wordy? Maybe the latter bit could go in presenter notes.

Reply:
Sounds reasonable.